apps/main/src/llamalend/features/borrow/useBorrowForm.tsx

```diff
@@ -63,7 +63,18 @@ export function useBorrowForm<ChainId extends IChainId>({
     reset: resetCreation,
   } = useCreateLoanMutation({ chainId, poolId: market?.id, reset: form.reset })
 
-  const { borrowToken, collateralToken } = useMemo(() => market && getTokens(market, chain), [market, chain]) ?? {}
+  const { borrowToken, collateralToken } =
+    useMemo(() => {
+      const tokens = market && getTokens(market)
+
+      return (
+        tokens && {
+          borrowToken: { ...tokens.borrowToken, chain },
```

Comment:
Should we consider the possibility of tokens in different chains?

Reply:
Yeah, sometimes tokens have the same address across chains